daemons/driver/requirements.txt

```diff
@@ -1,3 +1,3 @@
 requests==2.20.0
-dcplib==1.4.0
+dcplib==1.5.1
```

Comment:
We should consider using the root requirements.txt rather than individual requirements.txt files for these daemons. Or is this done to limit package size?

Reply:
Yup to limit package size. I'm blanking on which part of the deployment actually uses the root `requirements.txt` - can you remind me?

Reply:
Actually don't think any part of the deployments uses the root requirements.txt at the moment